OSS13 Client/Sources/Graphics/UI/Widget/Console.cpp

```diff
@@ -48,18 +48,20 @@ void Console::Update(sf::Time timeElapsed) {
 	if (copy_to_clipboard) ImGui::LogToClipboard();
 
 	for (const auto &row : rows) {
+		scrollToBottom = ImGui::GetScrollY() > ImGui::GetScrollMaxY() - 10.0f;
```

Comment:
Why 10f? We need something more strict here.

Reply:
Why not 10f?

Reply:
Why not not use magic numbers?

Reply:
GetScrollY and GetScrollMaxY return pixels. If user somehow scroll up for 9 pixels - scrolling will not be locked.

I propose to set 1 pixel here - any shift less then 1 pixel will not be visible.